oat-sa/tao-core-shared-libs-fe
# fix: add wordcount plugin to ckeditor to support IME characters

**Related to:** [TR-3112](https://oat-sa.atlassian.net/browse/TR-3112)

**Description:**
Japanese characters with the IME tool doesn't work well with the Latin characters counter, it needs a bytes counter.


**Changes:**

- Add a counter plugin to CKEditor that supports bytes counter to check limits with text from an IMO tool.

**How to check:**

- Install an IMO tool on your SO and enable the Japanese language to check it.
- Create a test with a TextExtendedInteraction, with plain text and other with Rich text.
- Set character limit constrain
- Write on the Textarea with the IMO and check if the limit is working.
